include/z64.h

```diff
@@ -1146,6 +1146,8 @@ typedef struct {
     /* 0x01E2 */ char unk_1E2[0x06];
 } TitleContext; // size = 0x1E8
 
+struct SelectContext;
```

Comment:
The struct also needs to be named in the typedef (aka `typedef struct SelectContext {` instead of `typedef struct {`, otherwise the compiler won't see them as the same thing.